docs/source/reference/router/configuration.mdx

```diff
@@ -589,17 +589,22 @@ Starting with v1.49.0, the router can run a Rust-native query planner. This nati
 
 <Tip>
 
-Starting with v1.57.0, to run the most performant and resource-efficient native query planner and to disable the V8 JavaScript runtime in the router, set the following options in your `router.yaml`:
+Starting with v1.59.0, the native query planner is GA and is run by default in the router with an ability to switch back to the now deprecated legacy implementation.
 
 ```yaml title="router.yaml"
-experimental_query_planner_mode: new
+experimental_query_planner_mode: legacy
 ```
 
-You can also improve throughput by reducing the size of queries sent to subgraphs with the following option:
+The legacy implementation will be removed in the next release.
+
+</Tip>
+
+<Tip>
+
+Previously, to run the most performant and resource-efficient native query planner and to disable the V8 JavaScript runtime in the router, set the following options in your `router.yaml`:
```

Comment:
Why are we leaving this in here?

Reply:
I was not entirely certain if we should remove it, since our docs don't correspond to specific versions of the router. For the time being I left it in, in case someone is on the older version of the router. Would you rather that we remove it?

Reply:
on second thought, i'll remove it.